README.md

```diff
@@ -31,22 +31,21 @@ once with:
 1. Open a second terminal and run Ardupilot:
 
     ```
-    ${ARDUPILOTDIR}/build/sitl/bin/arducopter-quad --model x
+    cd ${ARDUPILOTDIR}/build/sitl/bin
+    ./arducopter-quad --model x
     ```
 
 2. In a third terminal, run Mavproxy:
 
     ```
-    mavproxy.py --master tcp:127.0.0.1:5760 --out udp:127.0.0.1:14556 --streamrate -1
+    mavproxy.py --master tcp:127.0.0.1:5760 --out udp:127.0.0.1:14557 --streamrate -1
     ```
 
-3. Then simply open the following executable to run a simple test. This test
-connects to an active vehicle and displays its current variables every 3
-seconds.
+3. Then simply open the following executable to run a simple test.
 
     ```
-    build/tests/test
+    build/tests/tests
     ```
 
     This test connects to an active vehicle and displays its current state
-    variables every 3 seconds.
+    variables every once in a while.
```

Comment:
We could. But I noticed that I had changed this time interval to 1 second some time ago and forgot to fix the read-me. If someone else changes that value in the future, for any reason, it seems likely that the read-me will become outdated again and that no one will notice it. To avoid that problem, I think it's better just to get rid of the precision.